src/containers/pages/IRS/Map/helpers.ts

```diff
@@ -31,13 +31,16 @@ export const IRSIndicatorStops: { [key: string]: string[][] } = {
   ],
 };
 
+/** interface to describe function which processess an indicator property */
+export type IndicatorPropIngestor = (obj: any) => string | number;
+
 /** interface to describe and indicator row item */
 export interface IndicatorRowItem {
   denominator: string | number;
   description: string;
   numerator: string | number;
   title: string;
-  value: string | number;
+  value: string | number | IndicatorPropIngestor;
```

Comment:
@moshthepitt how do you feel about this approach of adding custom ingestion methods? Alternatively we can simplify it and just check if `focusArea[row.value] < 1` to handle this use case.

Reply:
@cKellyDesign I like this

Reply:
@moshthepitt I actually removed this for the sake of simplicity, i think it's safe to assume that:

1. All progress bar `value`s are represented as percentages
2. Values which are floats need to be multiplied by 100 and rounded
3. Values which are integers should be used as is
4. We never want to render decimals in the indicator row

Do you think these are safe assumptions? If so then we don't need to add the same ingestion method (with different strings) to all the `value` properties.

Reply:
@cKellyDesign I think these sound sound :)